dev-python/cadquery-ocp/cadquery-ocp-7.5.2_beta.ebuild

```diff
@@ -0,0 +1,172 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..9} )
+
+# OCP uses "python-single-r1" only because VTK uses "python-single-r1".
+inherit check-reqs cmake llvm multiprocessing python-single-r1
+
+MY_PN=OCP
+MY_PV="${PV//_/-}"
+MY_P="${MY_PN}-${MY_PV}"
+OCCT_PV=$(ver_cut 1-3)
+
+DESCRIPTION="Python wrapper for OCCT generated using pywrap"
+HOMEPAGE="https://github.com/CadQuery/OCP"
+SRC_URI="https://github.com/CadQuery/OCP/archive/refs/tags/${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+# CMake and VTK requirements derive from the "OCP/CMakeLists.txt" file
+# generated by the src_prepare() phase.
+BDEPEND=">=dev-util/cmake-3.16"
+RDEPEND="
+	${PYTHON_DEPS}
+	~sci-libs/opencascade-${OCCT_PV}[json,tbb,vtk]
+	>=sci-libs/vtk-9.0.0[${PYTHON_SINGLE_USEDEP}]
```

Comment:
Sorry for being late to the party. Didn't had the time yesterday to review this further. We need to add `[python]` USE flag here as well, i.e. `vtk-9.0.0[python,${PYTHON_SINGLE_USEDEP}]`.

Reply:
Added, Thanks :+1: